packages/web-react/src/components/Toast/stories/ToastBar.stories.tsx

```diff
@@ -1,7 +1,9 @@
 import { Markdown } from '@storybook/blocks';
 import type { Meta, StoryObj } from '@storybook/react';
 import React from 'react';
+import { EmotionColors } from '../../../constants';
 import { SpiritToastBarProps } from '../../../types';
+import { BasicToastColors } from '../constants';
```

Comment:
Elsewhere, the file `constants` is exported in the component index. It is missing in Toast (and maybe elsewhere too), and you import it here directly from the file, but elsewhere (Tag, for example) you import it from the index (`from '..'`). Can you please unify it?